aten/src/ATen/TensorIterator.cpp

```diff
@@ -292,16 +292,30 @@ void TensorIteratorBase::reorder_dimensions() {
 // See the [Common Dtype Computation] note
 ScalarType TensorIteratorBase::compute_common_dtype() {
   at::native::ResultTypeState state = {};
+  std::ostringstream input_dtypes;
```

Comment:
These changes don't seem necessary to support type promotion for addcmul and addcdiv?